.github/workflows/ci.yml

```diff
@@ -308,7 +308,7 @@ jobs:
           # We decided to exclude all external HTTP requests but the ones that under the domain greatexpectations.io
           # The reason is to avoid having network errors such as pages that throw 429 after too many requests (like Github)
           # and to prevent other possible errors related to user agent or lychee capturing hrefs from metadata that don't resolve to a specific page (preconnects in JS)
-          args: "--exclude='http.*' --include='^https://(.+\\.)?greatexpectations\\.io/' 'docs/docusaurus/build/**/*.html'"
+          args: "--exclude='http.*' 'docs/docusaurus/build/**/*.html'"
```

Comment:
**Note for reviewers:** I'm temporarily changing the link checker config to work around a [bug](https://greatexpectations.atlassian.net/browse/DOC-979) where the link checker fails when a PR contains a link to a page that that same PR is introducing. I'll undo this change after merging this PR to restore link checking